File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
     let a=new Set(nums);
    let j=0;
    for(let i of a){
        nums[j++]=i;
    }
     return  a.size;
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] a) {
    int n=a.length;
        if(n==1){
            return 1;
        }
        
        int i=0;
        int j=0;
        while(j<n){
             if(a[i]!=a[j]){
                a[++i]=a[j];
                j++;
            }
            else{
                j++;
            }
        }
       return i+1;
}
}